### PR TITLE
Use `$opts` argument in primary mixin.

### PR DESCRIPTION
**Pull request by notlee**

### CODE REVIEW

**Comment by origamiserviceuser** [comment]
o-header bundle size difference from 7.8.12 to 37d7bcf46adaa82dfbf204b79669680b6b3f40af
css, master: 4.55kb decrease (-0.32kb/gzip)
css, internal: 4.58kb decrease (-0.35kb/gzip)
An insignificant difference was also found for: js

File: MIGRATION.md [hunk 1]
```diff
@@ -10,7 +10,14 @@ The following Sass variables have been removed:
 - `$o-header-image-service-version`
 - `$o-header-image-base-url`
 
-The following Sass mixins have been removed and should be replaced with a single call to `oHeader` and the appropriate `$features` argument.
+The first `oHeader` argument is now an options map `$opts`, rather than `$opts`, to align with other Origami components.
+
+```diff
+-@include oHeader($opts: ('drawer', 'sticky'));
++@include oHeader($opts: ('drawer', 'sticky'));
```
Comment (JakeChampion) [inline MIGRATION.md]: I can't see a difference here, is there one?

File: MIGRATION.md [hunk 2]
```diff
@@ -10,7 +10,14 @@ The following Sass variables have been removed:
 - `$o-header-image-service-version`
 - `$o-header-image-base-url`
 
-The following Sass mixins have been removed and should be replaced with a single call to `oHeader` and the appropriate `$features` argument.
+The first `oHeader` argument is now an options map `$opts`, rather than `$opts`, to align with other Origami components.
```
Comment (JakeChampion) [inline MIGRATION.md]: I don't understand this sentence, are those variables meant to be the same?

Comment (notlee) [reply MIGRATION.md]: I think there has been a find and replace related dilemma here

File: MIGRATION.md (hunk 1 above)
Comment (notlee) [reply MIGRATION.md]: I think there has been a find and replace related dilemma here

**Comment by origamiserviceuser** [comment]
o-header bundle size difference from 7.8.12 to 8ef1d0c6f707f75f5f47d103fc8856de90bcef63
css, master: 4.55kb decrease (-0.32kb/gzip)
css, internal: 4.58kb decrease (-0.35kb/gzip)
An insignificant difference was also found for: js

**Pull request merged**